localtypings/pxtarget.d.ts

```diff
@@ -1190,9 +1178,13 @@ declare namespace pxt.tutorial {
         execute(options: CodeValidationExecuteOptions): Promise<CodeValidationResult>;
     }
 
+    interface CodeValidatorBaseProperties {
+        enabled?: string;
```

Comment:
Why is this a string and not a boolean? Does it have to do with getting the value from markdown?

Reply:
Yes, just matching the existing types for now; this is part of the other change I played around with but didn't commit, which would be to make it so basically all the webapp side does is link up parsing these into proper types and pass them to a validator object over in pxtlib; deferring doing that bit till we have a few more examples of validator types to work with so we can scope out what possible options / needs better and make sure it has the right api